flash_examples/integrations/labelstudio/video_classification.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import flash
+from flash.core.data.utils import download_data
+from flash.video import VideoClassificationData, VideoClassifier
+
+# 1 Download data
+download_data("https://label-studio-testdata.s3.us-east-2.amazonaws.com/lightning-flash/video_data.zip")
+
+# 1. Load export data
+datamodule = VideoClassificationData.from_labelstudio(
+    export_json="data/project.json",
+    data_folder="data/upload/",
+    val_split=0.2,
+    clip_sampler="uniform",
+    clip_duration=1,
+    decode_audio=False,
+)
+
+# 2. Build the task
+model = VideoClassifier(
+    backbone="slow_r50",
+    num_classes=datamodule.num_classes,
+)
+
+# 3. Create the trainer and finetune the model
+trainer = flash.Trainer(max_epochs=3)
+trainer.finetune(model, datamodule=datamodule, strategy="freeze")
+
+# 4. Make a prediction
+predictions = model.predict(os.path.join(os.getcwd(), "data/test"))
```

Comment:
Does LabeStudio comes with a mechanism to visualize the predictions too ?

Reply:
Maybe in the next PR ?

Reply:
> Does LabeStudio comes with a mechanism to visualize the predictions too ?

Yes, Label Studio has a mechanism to visualize the predictions, but it should be wrapped as a REST service